File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        current_count = 0
        max_ans = 0
        j = 0
        while j < len(nums):
            if nums[j] == 1:
                j+=1
                current_count+=1
                max_ans = max(max_ans,current_count)
            else:
                j+=1
                current_count = 0
        return max_ans

        

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int current_count = 0;
        int max_ans = 0;
        int j = 0;
        int n = nums.length;
        while(j < n)
        {
            if(nums[j] == 1)
            {
                j++;
                current_count+=1;
                max_ans = Math.max(max_ans,current_count);
            }
            else
            {
                j++;
                current_count = 0;
            }
        }
        return max_ans;
    }
}